pyproject.toml

```diff
@@ -53,7 +57,7 @@ boto3 = [
 ]
 
 dev = [
-    "artkit[openai, groq, anthropic, google-generativeai, huggingface, boto3, moto]",
```

Comment:
Why remove moto?

Reply:
I removed moto because it doesn't exist as a dependency here